orchestra/utils/time_tracking.py

```diff
@@ -131,3 +134,16 @@ def get_timer_current_duration(worker):
     if timer.start_time is None:
         return None
     return timezone.now() - timer.start_time
+
+
+def time_entry_hours_worked(today, worker, excluded_tasks=None):
+    if excluded_tasks is None:
+        excluded_tasks = []
+    aggregate_time = (
+        TimeEntry.objects
+        .filter(date=today)
+        .filter(worker=worker)
+        .exclude(assignment__task__in=excluded_tasks)
+        .aggregate(sum_time_worked=Sum('time_worked')))
+    total_duration = aggregate_time['sum_time_worked'] or timedelta(seconds=0)
```

Comment:
what's the field type of `time_worked`? I.e., how does the aggregate magically produce a timedelta?

Reply:
your lucky day :)

`time_worked = models.DurationField()`

which manifests as a `timedelta` when it reaches the ORM

Reply:
that's so fancy.